mysite/pages/views.py

```diff
@@ -30,21 +30,51 @@ def calendar(request):
 			check_websitemeta()
 			t = Task()
 			t.description_text = f'Untitled Task {total_tasks_ever_made()}'
+			t.time_estimate = 165 #Almost three hours
+			t.due_date = timezone.now() + timezone.timedelta(days = 10)
 			t.save()
+			update_subtasks(t)
 			total_tasks_ever_made(increment=1)
+			return HttpResponseRedirect("/calendar")
 
 		if ('delete_task' in request.POST): #If the form that we submitted has the name 'delete_task'
 			id_to_delete = request.POST['task_id'] #Get the ID of the task. This is stored in a input tag of type='hidden'
 			Task.objects.filter(id=id_to_delete).delete()
+			return HttpResponseRedirect("/calendar")
 
 		if ('edit_task' in request.POST): #If the form that we submitted has the name 'edit_task'
 			task_id = request.POST['task_id'] #Get the ID of the task. This is stored in a input tag of type='hidden'
 			return HttpResponseRedirect("task_"+task_id + "/edit_task")
 
 	course_list = Course.objects.all()
-	tsk_list = Task.objects.filter(due_date__lte=timezone.now()).order_by('-due_date')
+	tsk_list = Task.objects.all().order_by('-due_date')
 	return render(request, 'pages/tasks.html', {'tsk_list': tsk_list, 'course_list': course_list})
 
+def update_subtasks(task: Task):
```

Comment:
The meat and potatoes of this PR. The function gives a task a bunch of subtasks, spread out evenly between today and the day the task is due.